doc/sphinx/index.rst

```diff
@@ -20,9 +20,13 @@ In this documentation
 
 ..  grid:: 1 1 2 2
 
-   ..  grid-item:: :doc:`Tutorial <tutorial>`
+   ..  grid-item:: :doc:`Explore What Mir Is Capable of Using Mir Demos <explore-mir-using-demos>`
 
-       **Start here**: a hands-on introduction to Mir for new users
+       **Start here**: An introduction to what Mir can do using example demos
+
+   ..  grid-item:: :doc:`Using Mir Libraries to Build Your Own Wayland Compositor <make-your-own-compositor>`
+
+       **Start here**: An introduction to building Wayland compositors with Mir
```

Comment:
I think that this should be labeled as the main **Tutorial**. Mir is a product for building things, and the tutorial should be able building things with Mir :)

Reply:
Isn't it better to advertise what we're capable of (through the demo tutorial) before how we do it? i.e. make the user more excited to explore our product by showing them the cool features it has

Reply:
So from my point of view, if a user already knows what Mir does and just wants the code, the can just jump into the library tutorial. But I think most users will be unfamiliar with Mir so some showcase of its capabilities through demos might be useful

Reply:
Diataxis doesn't say there can be only one tutorial (that push comes from elsewhere - and might, if we push back, mean "at least one"), and I think we have two distinct lessons to offer.

### Tutorials

1. Learn what Mir can do
2. Learn how to develop a compositor